pkgs/servers/rmfakecloud/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "rmfakecloud";
+  version = "unstable-20220118";
```

Comment:
(hindsight 20/20: there's been a v0.0.6 release a few days ago, probably better to use than some random git rev; happy to upgrade after this gets in though)

Reply:
There's even a 0.0.7 now! I amended that to this PR.

Reply:
Cheers, can confirm new version also builds.